apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -77,7 +77,7 @@ class StageLockDialog extends React.Component {
         lockStatus: PropTypes.oneOf(Object.values(LockStatus)).isRequired
       })
     ),
-    selectedSectionId: PropTypes.string.isRequired,
```

Comment:
just double checking that removing `isRequired` was intentional?

Reply:
Yes, originally not selecting a section was represented by sectionId=''.  It is now represented by sectionId=null.  (You can see this handled on line 154 below.)